src/command/subgraph/mod.rs

```diff
@@ -19,20 +19,21 @@ pub struct Subgraph {
 
 #[derive(Debug, Serialize, StructOpt)]
 pub enum Command {
-    /// Check changes to an subgraph
+    /// Check for composition errors and breaking changes caused by an updated subgraph schema
+    /// against the federated graph and subgraph schemas in the Apollo graph registry
```

Comment:
i worry these are too many words in a similar way i worry that this command is... kinda hard to explain!

Reply:
We can cut "and subgraph schemas" to simplify slightly, yeah? If a change conflicts with another subgraph, it should also always conflict with the federated whole